numpy/core/umath.py

```diff
@@ -0,0 +1,25 @@
+from . import _multiarray_umath
+from numpy.core._multiarray_umath import *
+from numpy.core._multiarray_umath import _add_newdoc_ufunc, _arg
+
+__all__ = ['ERR_CALL', 'ERR_DEFAULT', 'ERR_IGNORE', 'ERR_LOG', 'ERR_PRINT',
```

Comment:
A comment / docstring would be good in this file too.

Reply:
added to both